provers/evm-prover/README.md

```diff
@@ -0,0 +1,29 @@
+# EVM Prover
+
+The EVM Prover is a gRPC service that generates zero-knowledge proofs for EVM state transitions. It is designed to work with IBC (Inter-Blockchain Communication) and specifically implements proofs compatible with the ICS-07 Tendermint client specification.
+
+## Usage
+
+> [!WARNING]
+> This gRPC service is still under development and may not work as described
+
+To run the server you will need to clone the repo and install rust and cargo. To run the node you also need to set the following environment variables:
+
+- `CELESTIA_NODE_AUTH_TOKEN` - the auth token for the celestia node you are connecting to.
```

Comment:
we have a node auth token file in test files so it should probably consume the key form there?

Reply:
yep